src/plugins/discourse/createGraph.js

```diff
@@ -173,19 +199,26 @@ class _GraphCreator {
       this.graph.addEdge(authorsTopicEdge(serverUrl, topic));
     }
 
+    const postDescriptions: Map<PostId, string> = new Map();
     for (const post of data.posts()) {
-      this.addPost(post);
+      const description = this.addPost(post);
+      postDescriptions.set(post.id, description);
     }
 
     for (const like of data.likes()) {
+      const postDescription =
+        postDescriptions.get(like.postId) || "[unknown post]";
+      this.graph.addNode(likeNode(serverUrl, like, postDescription));
       this.graph.addEdge(likesEdge(serverUrl, like));
+      this.graph.addEdge(createsLikeEdge(serverUrl, like));
     }
   }
 
-  addPost(post: Post) {
+  addPost(post: Post): string {
     const topicTitle =
       this.topicIdToTitle.get(post.topicId) || "[unknown topic]";
```

Comment:
Nit: returning just the post description here seems weird.
Prefer to factor out a `postDescription(post: Post): string` method, which both `addPost()` use, and the constructor's loop uses.

Reply:
I refactored it to be consistent with `topicIdToTitle`; the weird return is removed.